code/modules/mob/living/simple_animal/hostile/retaliate/cavern.dm

```diff
@@ -173,4 +173,7 @@
 
 /mob/living/simple_animal/hostile/retaliate/minedrone/fall_impact()
 	visible_message("<span class='danger'>\The [src] bounces harmlessly on its inflated wheels.</span>")
-	return FALSE
\ No newline at end of file
+	return FALSE
+
+mob/living/simple_animal/hostile/retaliate/minedrone/get_bullet_impact_effect_type(var/def_zone)
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/retaliate/minedrone/get_bullet_impact_effect_type(var/def_zone)
```

Reply:
That's wild it even compiled
I must've just been really unlucky during testing